presto-main/src/main/java/io/prestosql/metadata/TableLayoutResult.java

```diff
@@ -25,15 +25,22 @@
 
 public class TableLayoutResult
 {
+    private final TableHandle newTableHandle;
```

Comment:
Why do we need this field since we still have `layout`?

Reply:
Layout doesn't have a reference to the ConnectorTableHandle that's needed to construct the new TableHandle. I could add that, but it seemed conceptually cleaner to do it this way.